tools/mhr_smpl_conversion/README.md

```diff
@@ -0,0 +1,187 @@
+# SMPL-MHR Conversion Tool
+
+A Python toolkit for converting between SMPL/SMPLX and MHR body model representations. This tool provides bidirectional conversion capabilities with multiple optimization backends and identity handling options.
+
+## Overview
+
+The SMPL-MHR Conversion Tool enables seamless conversion between different 3D human body model formats:
+
+- **SMPL/SMPLX → MHR**: Convert SMPL/SMPLX model parameters/meshes to MHR format
+- **MHR → SMPL/SMPLX**: Convert MHR parameters back to SMPL/SMPLX format
+
+The tool uses barycentric interpolation for topology mapping and offers multiple optimization backends for parameter fitting.
+
+## Features
+
+### 🔄 **Bidirectional Conversion**
+- SMPL/SMPLX parameters or vertices → MHR parameters
+- MHR parameters or vertices → SMPL/SMPLX parameters
+
+### ⚙️ **Multiple Optimization Methods**
+- **PyMomentum**: CPU-based Gauss-Newton hierarchical optimization with robust fitting
+- **PyTorch**: GPU-accelerated optimization with edge and vertex loss
+
+### 🎭 **Identity Handling**
+- **Single Identity**: Use consistent shape parameters across all frames
+- **Multiple Identities**: Unique shape parameters for each frame
+
+### 📊 **Output Options**
+- Return meshes in trimesh.Trimesh format
+- Return parameter dictionaries
+- Return vertex as numpy arrays
+- Return fitting errors
+
+## Installation
+
+### Prerequisites
+
+```bash
+# On top of MHR
+pixi add --pypi trimesh scikit-learn tqdm smplx
+```
+
+### SMPL/SMPLX Model Files
+
+You'll need the official SMPL/SMPLX model files:
+
+1. **SMPL**: Download from [SMPL website](https://smpl.is.tue.mpg.de/)
+2. **SMPLX**: Download from [SMPLX website](https://smpl-x.is.tue.mpg.de/)
+
+**Note**: If you run into issues with the `.pkl` SMPL(X) model file, try the official `.npz` files instead.
+
+## Quick Start
+
+### Run Examples
+
+```bash
+pixi run python example.py --smpl path/to/smpl/model.pkl --smplx path/to/smplx/model.pkl -o output_dir
+```
+Three conversions will be conducted and the results will be exported in three folders under the output_dir. The results folders are named as {source_model}\_{input_format}2{target_model}\_{method}(_{if_single_identity_sequence}):
+
+```
+mhr_smpl_conversion/
+├── output_dir
+│   ├── smpl_para2mhr_pymomentum
+│   ├── mhr_para2smplx_pytorch_single_identity
+│   ├── smplx_mesh2mhr_pytorch_single_identity
+└──...
+```
+
+Meshes of the source and the target model can be found under each conversion results folder.
+
+### Programmatic Usage
+
+```python
+import torch
+from mhr.mhr import MHR
+from smpl_mhr import Conversion
```

Comment:
Does this work? I cannot find any smpl_mhr py file or library, but maybe I didn't install it properly.